src/main/java/digital/slovensko/autogram/ui/gui/SigningDialogController.java

```diff
@@ -73,6 +82,41 @@ public void onChangeKeyButtonPressed(ActionEvent event) {
         autogram.pickSigningKeyAndThen(gui::setActiveSigningKey);
     }
 
+    public void onShowPresentSignaturesButtonPressed(ActionEvent event) {
+        if (presentSignaturesDialogController == null)
+            presentSignaturesDialogController = new PresentSignaturesDialogController(visualization.getJob());
+
+        var root = GUIUtils.loadFXML(presentSignaturesDialogController, "present-signatures-dialog.fxml");
+
+        var stage = new Stage();
+        stage.setTitle("Prítomné podpisy");
+        stage.setScene(new Scene(root));
+        stage.setResizable(false);
+        stage.initModality(Modality.APPLICATION_MODAL);
```

Comment:
Preco modal? Chceme znefunkcnit applikaciu ked sa toto jedno okno otvori?

Reply:
Jasné. To je vlastne nechtiac modal.